include/multipass/utils.h

```diff
@@ -110,8 +111,9 @@ std::string match_line_for(const std::string& output, const std::string& matcher
 
 // virtual machine helpers
 bool is_running(const VirtualMachine::State& state);
-void wait_until_ssh_up(VirtualMachine* virtual_machine, std::chrono::milliseconds timeout,
-                       std::function<void()> const& ensure_vm_is_running = []() {});
+void wait_until_ssh_up(
+    VirtualMachine* virtual_machine, std::chrono::milliseconds timeout,
+    std::function<void()> const& ensure_vm_is_running = []() {});
```

Comment:
This looks like a `clang-format` change?  If so, please only run `clang-format` on the diff, not the whole file.  Unfortunately, it adds changes that really don't need to be reviewed.  Thanks!